apis/v0/plugin_interface.go

```diff
@@ -39,12 +39,22 @@ var Handshake = plugin.HandshakeConfig{
 	MagicCookieValue: handshakeCookieValue,
 }
 
+// PluginMap is the map of plugins we can dispense.
+var PluginMap = map[string]plugin.Plugin{
```

Comment:
I'm not sure this should be public - we don't want anyone overridding the pluginmap outside of the package.

Reply:
Good point!!! I have moved it to plugin_manager and make it a package private variable now. Seems we need to adjust this variable every time a new plugin is support.